filetracker/servers/migration.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import email.utils
+import os.path
+from hmac import new
+from uu import encode
+
+from six.moves.urllib.parse import parse_qs
+
+from filetracker.servers import base
+from filetracker.servers.files import FileTrackerServer
+from filetracker.servers.storage import FileStorage
```

Comment:
Most of the imports above are unused.

Reply:
Sorry, mistake from copy-pasting. Wasn't using PyCharm with this commit and forgot to check for such things.